pylcm/__init__.py

```diff
@@ -90,6 +90,7 @@ def msg_to_dict(  # noqa: C901
     verbose=False,
     lcm_timestamp=-1,
     decompress_jpeg=True,
+    image_dimensions=(640, 480),
```

Comment:
this doesn't get applied to RGB images. only depth data as binary str needs a known shape. so call it `depth_dimensions`.

Reply:
fixed